### DISCUSSION TITLE
Goal alerts on multiple chrome profiles

### DISCUSSION THREAD

**Discussion by thebigtine** [dailydotdev/daily#319] [Feature Requests]
For each business/major client partnership I'm involved in, I have a different chrome profile. I have daily.dev installed on all of them of course, but it can get irritating when every time I open a new tab on a new profile the same goal alerts pop up. Would be great to have it so once you have been alerted it wont popup in any other chrome profile (or anywhere you are logged in with the same account). 

Thanks!

**Comment by idoshamun** [comment]
That makes perfect sense! I will add it to our backlog :)

**Reply by thebigtine** [reply]
Awesome!
